packages/react/src/components/SignInButton.tsx

```diff
@@ -11,21 +12,27 @@ export const SignInButton = withClerk(({ clerk, children, ...props }: WithClerkP
   const child = assertSingleChild(children)('SignInButton');
 
   const clickHandler = () => {
-    const opts = {
+    const opts: SignInProps = {
+      forceRedirectUrl,
+      fallbackRedirectUrl,
       signUpFallbackRedirectUrl,
       signUpForceRedirectUrl,
-      signInForceRedirectUrl: forceRedirectUrl,
-      signInFallbackRedirectUrl: fallbackRedirectUrl,
     };
 
     if (mode === 'modal') {
       return clerk.openSignIn(opts);
     }
-    return clerk.redirectToSignIn(opts);
+    return clerk.redirectToSignIn({
+      ...opts,
+      signInFallbackRedirectUrl: fallbackRedirectUrl,
+      signInForceRedirectUrl: forceRedirectUrl,
```

Comment:
when calling the redirect methods, the prefixed props are necessary. Added here.